examples/kops-new-cluster/base/cluster.yaml

```diff
@@ -74,7 +74,7 @@ metadata:
     kops.k8s.io/cluster: aws-external-cloud-provider-example.k8s.local
   name: master-us-west-2a
 spec:
-  image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20200716
```

Comment:
Wondering if this one can be skipped altogether. kOps should use the latest stable if this one is not specified.

Reply:
I think it should come from channels, or we can add the ssm way:
```
image: ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id
```

Reply:
Sure... let me test these options and see

Reply:
Updated to use ssm parameter.

Reply:
@olemarkus can you review again when you get chance. I had addressed the comments.